src/unordered.rs

```diff
@@ -511,6 +511,16 @@ pub struct Plan {
     n: usize,
 }
 
+impl core::fmt::Debug for Plan {
+    fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
+        f.debug_struct("Plan")
+            .field("base_algo", &self.base_algo)
+            .field("base_size", &self.base_n)
```

Comment:
should be `field("base_n", &self.base_n)` ?

Reply:
yep bad copy paste thanks !